indico/modules/events/abstracts/controllers/display.py

```diff
@@ -44,3 +46,29 @@ def _process(self):
         pdf = AbstractToPDF(self.abstract)
         file_name = secure_filename('abstract-{}.pdf'.format(self.abstract.friendly_id), 'abstract.pdf')
         return send_file(file_name, pdf.generate(), 'application/pdf')
+
+
+class RHDisplayReviewableTracks(RHConferenceBaseDisplay):
+    def _process(self):
+        user = session.user
+        tracks_set, events_set = set(), set()
+        tracks_set.update(user.convener_for_tracks, user.abstract_reviewer_for_tracks)
+        events_set.update(user.global_convener_for_events, user.global_abstract_reviewer_for_events)
+        for e in events_set:
+            tracks_set.update(e.tracks)
+        tracks = {t for t in tracks_set if t.event_id == self.event_new.id}
```

Comment:
Actually you can already filter it like this above:

    tracks_set = (user.convener_for_tracks | user.abstract_reviewer_for_tracks) & set(event.tracks)